### ISSUE TITLE
Improve help command

### ISSUE THREAD

**Issue by Th3J0nny** [Th3J0nny/discord-all-in-bot#3]
For each feature/command, add an explanation. 

See: https://discordpy.readthedocs.io/en/stable/ext/commands/api.html?highlight=help#help-commands

**Comment by Sajid2001** [comment]
I am a little confused on how to do this.

**Comment by Th3J0nny** [comment]
What exactly are you confused about?

**Comment by Sajid2001** [comment]
Where do I add the explanation and how do I add it? And what is the discord.py documentation link for?

**Comment by Th3J0nny** [comment]
While looking into it, I realized, that this is probably not as easy as a first issue as I thought it was^^

We need to think about what the best way is to improve the default help command. Maybe one possibility is, to set the `help_command` when running the bot to `None` (https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.help_command) and include a new feature "Help" which calls each feature to get information about each command. Ideally, the Help class will automatically iterate through all classes in the feature directory, while all features implement a `get_help_text(command_name)` function. 
Alternatively, we can use the discord-pretty-help library, but I didn't look into it and thus don't know how it fits with our code structure. (modularity) https://github.com/stroupbslayen/discord-pretty-help

**Comment by Th3J0nny** [comment]
Either, we can add a description in the `@commands.command` annotation of each command like [here](https://github.com/stroupbslayen/discord-pretty-help/blob/master/tests/test_discord_pretty_help.py) 
Or, we can implement the library [discord-pretty-help](https://github.com/stroupbslayen/discord-pretty-help)

EDIT: For the pretty-help library, we'd need to upgrade the permissions of the bot to include `manage-messages`.

**Comment by Sajid2001** [comment]
I think I like the first method better. So all we need to do is just add that description to the commands, right? That should not be too difficult for anyone. I also did try to make a language translator feature in the bot. I have used discordJS a little bit in the past, but I am still new to discord.py, so the feature may not work very well. Is it possible for me to have permission to push the code onto this repository?

**Comment by Th3J0nny** [comment]
I agree, the first method would is more feasible for now. You should be able to open up a pull request for your branch and push your code. I can then review the code. If not, I'll have to edit the repository permissions. This is my first public repo as well :D
